Cython/Includes/cpython/datetime.pxd

```diff
@@ -161,11 +255,34 @@ cdef inline object datetime_new(int year, int month, int day, int hour, int minu
 cdef inline object timedelta_new(int days, int seconds, int useconds):
     return PyDateTimeAPI.Delta_FromDelta(days, seconds, useconds, 1, PyDateTimeAPI.DeltaType)
 
+# Create datetime object using PEP 495 constructor.
+cdef inline object datetime_and_fold_new(int year, int month, int day, int hour, int minute, int second, int microsecond, int fold):
+    return PyDateTime_FromDateAndTimeAndFold(year, month, day, hour, minute, second, microsecond, fold)
+
+# Create time object using PEP 495 constructor.
+cdef inline object time_and_fold_new(int hour, int minute, int second, int usecond, int fold):
+    return PyTime_FromTimeAndFold(hour, minute, second, usecond, fold)
+
+cdef inline object timezone_new(int offset, str name=''):
+    return PyTimeZone_FromOffsetAndName(offset, name)
```

Comment:
I don't know where you got the information from that `offset` is an `int`. According to the implementation, it's a `timedelta` instance.